atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/RangeRequest.java

```diff
@@ -292,19 +289,22 @@ public boolean isReverse() {
          * This will set the start and the end to get all rows that have a given prefix.
          */
         public Builder prefixRange(byte[] prefix) {
+            Preconditions.checkNotNull(prefix, "prefix cannot be null");
```

Comment:
This doesn't actually change any behaviour as both branches of the `if` statement check this on the first line.

I added this for more clarity because looking at the code, you can see `prefix.clone()` will fail with an NPE but you don't see that `createNamesForReversePrefixScan` will fail without diving into that method.